mask/eolearn/mask/cloud_mask.py

```diff
@@ -292,3 +297,563 @@ def get_s2_pixel_cloud_detector(threshold=0.4, average_over=4, dilation_size=2,
                                 average_over=average_over,
                                 dilation_size=dilation_size,
                                 all_bands=all_bands)
+
+
+# Twin classifier
+MONO_CLASSIFIER_NAME = 'pixel_s2_cloud_detector_lightGBM_v0.1.joblib.dat'
+MULTI_CLASSIFIER_NAME = 'ssim_s2_cloud_detector_lightGBM_v0.2.joblib.dat'
+
+class AddMultiCloudMaskTask(EOTask):
+    """ This task wraps around s2cloudless and the SSIM-based multi-temporal classifier.
+    Its intended output is a cloud mask that is based on the outputs of both
+    individual classifiers (a dilated intersection of individual binary masks).
+    Additional cloud masks and probabilities can be added for either classifier or both.
+
+    Prior to feature extraction and classification, it is recommended that the input be
+    downscaled by specifying the source and processing resolutions. This should be done
+    for the following reasons:
+        - faster execution
+        - lower memory consumption
+        - noise mitigation
+
+    Resizing is performed with linear interpolation. After classification, the cloud
+    probabilities are themselves upscaled to the original dimensions, before proceeding
+    with masking operations.
+    """
```

Comment:
given the high functionality of the task, it might be good to add to docstring a couple of call examples for running either or both single and MT

Reply:
Added an example for combined masks (mono + multi) and only mono masks.